narwhals/_dask/expr.py

```diff
@@ -272,6 +272,19 @@ def round(self, decimals: int) -> Self:
             lambda _input, decimals: _input.round(decimals), "round", decimals
         )
 
+    def all(
+        self,
+        *,
+        ignore_nulls: bool = True,
+    ) -> Self:
+        return self._from_call(
+            lambda _input, ignore_nulls: _input.all(
+                axis=None, skipna=ignore_nulls, split_every=False, out=None
+            ),
+            "all",
+            ignore_nulls,
+        )
```

Comment:
👌👌

Reply:
After making the adjustments to the test as you described, I ran the tests and received the error:
`FAILED tests/expr_and_series/any_all_test.py::test_any_all[dask_lazy_constructor] - AttributeError: 'Scalar' object has no attribute 'name'`

I had thought it may have something to do with the fact that `Expr.all()` might not work for series, but on viewing re-reading the error I don't think that is the case. Please, what am I missing? I do not understand which object it is referring to as a `scalar` object.

Reply:
Oh I see, this is related to #694 and Marco is working on a partial fix in #723. 

The scalar is a [Dask native Scalar object](https://github.com/dask/dask-expr/blob/1c646712b3c74eb9bed52bec59e442ce24d165c8/dask_expr/_collection.py#L4761). 

Maybe this needs to wait until #723 is merged (or we need to find another way of testing, but I would not suggest it)

Reply:
@mistShard heads up #723 has been merged! By introducing `returns_scalar=True` you should be able to make this work 🙌🏼